releases/7.6.0.md

```diff
@@ -170,7 +172,7 @@ Minor incompatibilities:
     is now a more attractive target for overriding than `run_load_task()`.
 -   `unzip_file()` moved from `arches.setup` to `arches.app.utils.zip`
 -   Version-related utils moved from `arches.setup` to `arches.version`
-
+-   `arches.app.utils.compatibility` was removed in favor of a Django system check.
```

Comment:
Maybe we should also mention that the settings MIN_ARCHES_VERSION and MAX_ARCHES_VERSION are no longer in use.